### ISSUE TITLE
wtrace not working in Docker

### ISSUE THREAD

**Issue by lowleveldesign** [lowleveldesign/wtrace#6]
wtrace fails when run inside a Docker container. Example code to replicate the issue:

```
C:\Users\Sebastian>docker inspect -f "{{ .NetworkSettings.Networks.nat.IPAddress }}" my-running-site
172.23.31.132
 
PS samplesite> docker run -d -v d:\temp:c:\host --name my-running-site aspnet-site
14065bc4473a0857b246e7d1812a2b338f4597bd980d8b193f4edfa95be2320e
 
PS samplesite> docker ps -all
CONTAINER ID        IMAGE               COMMAND                   CREATED             STATUS              PORTS               NAMES
14065bc4473a        aspnet-site         "C:\\ServiceMonitor..."   36 seconds ago      Up 20 seconds       80/tcp              my-running-site
 
PS samplesite> docker exec -it 140 powershell
 
PS C:\wtrace\wtrace> .\wtrace.exe cmd
ERROR: severe error happened when starting application: Value does not fall within the expected range.
```

**Comment by lowleveldesign** [comment]
Should be possible based on this blogpost: https://blogs.msdn.microsoft.com/vancem/2018/08/16/using-perfview-etw-in-windows-docker-conainers/

**Comment by lowleveldesign** [comment]
Fixed in 3.0. Requires a Windows container with full .NET
